manifests/Arion-Air Superiority.yaml

```
@@ -20,3 +20,7 @@ description: |
   Special thanks to 1010todd, yuyuko_saigyouji. and gefulltetaubenbrust for some of the sprites.
 url: https://github.com/Spectre907YT/Arion-Air_Superiority/archive/refs/tags/3.0.1.zip
 iconUrl: https://github.com/Spectre907YT/Arion-Air_Superiority/raw/3.0.1/icon.png
+autoupdate:
+  type: tag
+  url: https://github.com/Spectre907YT/Arion-Air_Superiority/releases/download/$version/Arion-Air_Superiority-$version.zip
```

This will require you to upload a release asset called `Arion-Air_Superiority-3.0.3.zip` to the latest tag (note the lack of `v`). If that's not desired you can continue to use github auto-generated archives:

```suggestion
  url: https://github.com/Spectre907YT/Arion-Air_Superiority/archive/refs/tags/$version.zip
```

---

I added a new zip file for the latest ver, should i still edit it?